data/json/monster_weakpoints/migo_weakpoints.json

```
@@ -0,0 +1,90 @@
+[
+  {
+    "type": "weakpoint_set",
+    "id": "wps_mi-go",
+    "weakpoints": [
+      {
+        "id": "arm",
+        "name": "the arm",
+        "crit_mult": { "all": 0.75 },
+        "difficulty": { "ranged": 4, "melee": 3 },
+        "coverage_mult": { "point": 0.75 },
+        "effects": [
+          { "effect": "staggered", "chance": 15, "message": "The %s is knocked off-balance!", "damage_required": [ 10, 30 ] },
+          {
+            "effect": "staggered",
+            "chance": 25,
+            "message": "The %s is knocked off-balance!",
+            "damage_required": [ 31, 100 ]
+          }
+        ],
+        "coverage": 6
+      },
+      {
+        "id": "leg",
+        "name": "the leg",
+        "crit_mult": { "all": 0.75 },
+        "difficulty": { "ranged": 4, "melee": 3 },
+        "coverage_mult": { "point": 0.75 },
+        "effects": [
+          { "effect": "staggered", "chance": 15, "message": "The %s is tripped off-balance!", "damage_required": [ 10, 30 ] },
```

```suggestion
          { "effect": "staggered", "chance": 10, "message": "The %s is tripped off-balance!", "damage_required": [ 15, 30 ] },
```

Mi-go have more legs and should be harder to trip.